### PR TITLE
Display baselayers with radio buttons

### LINKED ISSUE #143: Use Radio Buttons for Base Layers
Currently base layers use check boxes even though only one can be active at a time. Radio buttons would make more sense here.

image

### PR DESCRIPTION
**Pull request by jansule**
BaseLayers will now be displayed with radio buttons instead of checkboxes. Uses property `isBaseLayer` for distinguishing baseLayers from overlays.

image

fixes #143

### CODE REVIEW

File: app/view/LayerTree.js [hunk 1]
```diff
@@ -158,6 +158,16 @@ Ext.define('CpsiMapview.view.LayerTree', {
             });
             me.setStore(groupedLayerTreeStore);
 
+            me.getRootNode().cascade(function (node) {
+                var data = node.getData();
+                if (data.leaf) {
+                    var isBaseLayer =  data.get('isBaseLayer');
+                    if (isBaseLayer) {
```
Comment (chrismayer) [inline app/view/LayerTree.js]:
```suggestion
                    if (data.leaf && data.get('isBaseLayer')) {
```
surely not blocking

Comment (chrismayer) [reply app/view/LayerTree.js]: What I wanted to say is to merge the 2 if statements to one.

Comment (jansule) [reply app/view/LayerTree.js]: done in https://github.com/compassinformatics/cpsi-mapview/pull/189/commits/f7daaa60823018db884ce2a4059da2b4903d2fd2

**Comment by geographika** [comment]
Thanks @jansule !

**Pull request merged**